tests/conftest.py

```diff
@@ -27,34 +28,42 @@ def app():
 
 
 @pytest.fixture(scope="function")
-def admin_setup(base=app):
-    """Login as an admin"""
-    base.go_to_site()
-    base.landing.sign_up_btn.click_btn_by_css()
-    base.modal.login(config.ADMIN_EMAIL, config.ADMIN_PASS)
-    return base
+def admin_setup(app):
+    """
+        Login as an admin
+    """
+    app.go_to_site()
+    app.landing.sign_in_up_btn.click_btn_by_css()
+    app.modal.login(config.ADMIN_EMAIL, config.ADMIN_PASS)
+    return app
 
 
 @pytest.hookimpl(tryfirst=True, hookwrapper=True)
 def pytest_runtest_makereport(item):
-    """Hook the "item" object on a test failure"""
+    """
+        Hook the "item" object on a test failure
+    """
```

Comment:
Shorter docstring would be better - https://www.datacamp.com/tutorial/docstrings-python